### ISSUE TITLE
💡 New community highlights page

### ISSUE THREAD

**Issue by TomWFox** [parse-community/parse-community.github.io#44]
**Is your feature request related to a problem? Please describe.**
The [community highlights section](https://parseplatform.org/#community) is not particularly easy to navigate, there is little to distinguish different types of projects, no indication of activity etc - the section only gets worse as more items are added.

**Describe the solution you'd like**
A completely separate page would probably work best so that it could be expanded without making the front-page overly long. It would be good to categories all the projects, add platform logos, show the number of stars (if on GitHub) and add a search function.

I think https://modules.nuxtjs.org could provide some good inspiration/direction - they include both official and community modules - this might be worth considering.

**Comment by mtrezza** [comment]
Good idea, it could also solve issue https://github.com/parse-community/parse-server/issues/7120.
